packages/blockchain-wallet-v4-frontend/src/modals/Interest/DepositForm/template.success.tsx

```diff
@@ -63,12 +52,13 @@ import {
   ToggleCoinText,
   ToggleFiatText,
   Top,
-  TopText
+  TopText,
 } from './model'
 import TabMenuTimeFrame from './TabMenuTimeFrame'
 import { maxDepositAmount, minDepositAmount } from './validation'
 
-const DepositForm: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
+// eslint-disable-next-line
```

Comment:
What's wrong here?

Reply:
actually I can fix it, missing proper pass to `{}`